diff --git a/Makefile b/Makefile
index d32d874..1b155b4 100644
--- a/Makefile
+++ b/Makefile
@@ -49,8 +49,8 @@ envtools:
 install:
 	install $(OUTPUT) $(INSTALLPATH)
 clean:
-	find  $(SRC) -type f -executable -exec rm {} \;
-	find  $(TEMPL_SRC) -type f -executable -exec rm {} \;
+	find  $(SRC) -type f -perm /111 -exec rm {} \;
+	find  $(TEMPL_SRC) -type f -perm /111 -exec rm {} \;
 	rm -f $(TEMPL_SRC)/*.o
 	rm -f $(TEMPL_SRC)/*.so
 	rm -f $(SRC)/*.o
